app/uk/gov/hmrc/slacknotifications/services/NotificationService.scala

```diff
@@ -132,27 +141,23 @@ class NotificationService @Inject()(
 
   private def withExistingRepository[A](
```

Comment:
since these function no longer takes a callback to do something `with` the value , can it be renamed to `get`/`find` or even just drop the prefix?

Reply:
Renamed with `get` prefix